sys/auto_init/saul/auto_init_isl29020.c

```diff
@@ -49,13 +54,12 @@ extern saul_driver_t isl29020_saul_driver;
 
 void auto_init_isl29020(void)
 {
-    for (unsigned int i = 0; i < ISL29020_NUM; i++) {
-        const isl29020_params_t *p = &isl29020_params[i];
+    assert(ISL29020_NUM == ISL29020_INFO_NUM);
```

Comment:
Just wondering why you used `==` and not `>= `?

Reply:
Because the loop below is accessing 2 different lists, one for the list of device descriptor and one for list device information: each list length must match.